charts/pact-broker/values.yaml

```diff
@@ -337,6 +337,18 @@ broker:
     # -- Success threshold for readinessProbe
     successThreshold: 1
 
+  # Volumes to mount
+  volumes: []
+    # - hostPath:
```

Comment:
We should probably use a different example here as `hostPath` is [advised against](https://kubernetes.io/docs/concepts/storage/volumes/#hostpath). How about [`emptyDir`](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir-configuration-example)?

```yaml
  - name: cache-volume
    emptyDir:
       sizeLimit: 500Mi
```

Reply:
@ChrisJBurns, sorry for the delay on my part as well, holidays ...